en-US/Docs/PinMappingsRPi.md

```diff
@@ -45,6 +46,8 @@ The following GPIO pins are accessible through APIs:
 | 11    | PullDown      | SPI0 SCLK           | 23                 |
 | 12    | PullDown      |                     | 32                 |
 | 13    | PullDown      |                     | 33                 |
+| 14    | PullUp        | UART0 TXD           | 8                  |
```

Comment:
We don't expose pins 14/15 as GPIO. They are permanently assigned to UART. Please remove this change and I will approve the PR.